bundles/org.openhab.binding.hdpowerview/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -47,4 +47,33 @@
 		</config-description>
 	</bridge-type>
 
+	<bridge-type id="gen3">
```

Comment:
Since Gen 3 is a whole line of new products, probably we should name the bridge according to its type rather than generation alone. I would suggest either:
```suggestion
	<bridge-type id="gateway">
```

or:
```suggestion
	<bridge-type id="gateway3">
```
(which would be consistent with "shade3")

Reply:
Yes.